src/components/Icons.jsx

```diff
@@ -169,6 +172,9 @@ export const SettingIcon = Wrapper(Edit);
 export const ArrowIcon = Wrapper(Arrow);
 export const FavoriteIcon = Wrapper(Star);
 export const DownloadIcon = Wrapper(Download);
+export const CalendarIcon = Wrapper(Calendar);
+export const TimeIcon = Wrapper(Time);
+export const CloseCircleIcon = Wrapper(CloseCircle);
```

Comment:
we only want these 3 icons in the app. What you can do is export out `Wrapper` instead (it could use a better name), and import those svg in the app.